### ISSUE TITLE
poppler 20.12 no longer installs poppler-splash.pc

### ISSUE THREAD

**Issue by foutrelis** [jwilk/pdf2djvu#144]
pdf2djvu looks for `poppler-splash.pc` (I'm assuming) in order to ensure poppler is built with the Splash backend. Unfortunately poppler 20.12 doesn't include this .pc file because it's considered a private backend. [1] [2]

This breaks building pdf2djvu with the latest poppler, unless I modify the configure script to look for `poppler.pc` instead of `poppler-splash.pc`. The alternative of checking for `SplashOutputDev.h` comes to mind, but perhaps there's a better way to restore compatibility with poppler 20.12.

[1] https://lists.freedesktop.org/archives/poppler/2018-February/012808.html
[2] https://gitlab.freedesktop.org/poppler/poppler/-/issues/982

**Comment by badshah400** [comment]
Just replacing the `poppler-splash` check with `poppler` does it for me. Here's a patch:

```
Index: pdf2djvu-0.9.17.1/configure.ac
===================================================================
--- pdf2djvu-0.9.17.1.orig/configure.ac
+++ pdf2djvu-0.9.17.1/configure.ac
@@ -73,8 +73,8 @@ do
 done
 AC_MSG_RESULT([ok])
 
-PKG_CHECK_MODULES([POPPLER], [poppler-splash >= 0.35.0])
-poppler_version=$($PKG_CONFIG --modversion poppler-splash)
+PKG_CHECK_MODULES([POPPLER], [poppler >= 0.35.0])
+poppler_version=$($PKG_CONFIG --modversion poppler)
 AC_DEFINE_UNQUOTED([POPPLER_VERSION_STRING], ["$poppler_version"], [Define to the version of Poppler])
 parse_poppler_version()
 {
```

**Comment by CouldBeThis** [comment]
Hello, I have been trying to figure out the presumably simple steps required to apply the above text to this application. 
I've been working on this for a few hours (non programer here!), using as my original files  [pdf2djvu-0.9.17.1.tar.xz](https://github.com/jwilk/pdf2djvu/releases/tag/0.9.17.1). 

1. Started with a clean copy of files 
2. Copied text [above](https://github.com/jwilk/pdf2djvu/issues/144#issuecomment-756471768) (including trailing line-break) into a file `patch.diff` in the root directory of project
3. Perusing the [manpage](https://linux.die.net/man/1/patch) I see there are different configurations for running `patch` and I tried a couple but the best seemed to be just `patch < patch.diff` 
4. Since it looks like the patch just modifies 2 lines in `configure.ac`,  checked that this was done properly. 
5. However it does not prevent the error:

```zsh
../configure
checking for grep that handles long lines and -e... /usr/local/bin/ggrep
checking for egrep... /usr/local/bin/ggrep -E
checking for a BSD-compatible install... /usr/local/bin/ginstall -c
checking for g++... g++
checking whether the C++ compiler works... yes
checking for C++ compiler default output file name... a.out
checking for suffix of executables...
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C++ compiler... yes
checking whether g++ accepts -g... yes
checking how to run the C++ preprocessor... g++ -E
checking whether we are using the GNU C++ compiler... (cached) yes
checking whether g++ accepts -g... (cached) yes
checking whether g++ supports C++11... no
checking whether g++ -std=gnu++11 supports C++11... yes
checking for special C compiler options needed for large files... no
checking for _FILE_OFFSET_BITS value needed for large files... no
checking for g++ option to support OpenMP... unsupported
checking for MinGW ANSI stdio... no
checking for pkg-config... /usr/local/bin/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for ddjvuapi >= 3.5.25... yes
checking DjVuLibre fitness... ok
checking for poppler-splash >= 0.35.0... no
configure: error: Package requirements (poppler-splash >= 0.35.0) were not met:

No package 'poppler-splash' found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables POPPLER_CFLAGS
and POPPLER_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
```

I notice the file `configure` has 12 instances of `poppler-splash` and I think that's the source of the above. I obviously tried doing a find/replace but it broke in other ways. (Sorry I can't find the output.) 

So I am probably doing the wrong thing. A hint would be appreciated as to exactly what is meant to be happening.

**Comment by badshah400** [comment]
After applying the patch. but before running the `configure` script you need to run `autoreconf -fvi` from the dir where the `configure` script is, to incorporate the changes in `configure.ac` into the `configure` script. Thereafter, the usual `./configure`, `make`, `make install` should work.

**Comment by jwilk** [comment]
Fixed in [0.9.18](https://github.com/jwilk/pdf2djvu/releases/tag/0.9.18).
